src/main/resources/application.yml

```diff
@@ -15,8 +15,8 @@ logging:
   profile: DEV
 
 liquibase:
-  user: actionsvc
-  password: actionsvc
+  user: postgres
```

Comment:
This might have impact on people running it locally and not in docker like @pricem14pc

Reply:
Fixed